chapters/ch08.md

```diff
@@ -19,18 +25,25 @@ Here, neither classifier is obviously superior, so it doesn’t immediately guid
 
 ->
 
-| Classifier  | Percision | Recall | F1 score |
-| ----- | -------: | -------: | -----: |
-| A  | 95%  | 90% | 92.4% |
+| Bộ Phân Loại  | Percision | Recall |
+| ----- | -------: | -------: |
+| A  | 95%  | 90% |
+| B  | 98%  | 85% |
+
+Ở đây, không thuật toán nào tốt hơn một cách rõ ràng, vì vậy dựa vào kết quả này ta không thể ngay lập tức chọn ra một bộ phân loại tốt hơn.
 
 During development, your team will try a lot of ideas about algorithm architecture, model parameters, choice of features, etc. Having a **​single-number evaluation metric​**​ such as accuracy allows you to sort all your models according to their performance on this metric, and quickly decide what is working best.
 
 -> 
 
+Trong quá trình phát triển, nhóm của bạn sẽ thử rất nhiều ý tưởng liên quan đến cấu trúc thuật toán, tham số mô hình, lựa chọn các đặc trưng, v.v.. Việc có một **phép đo đơn trị** như độ chính xác sẽ giúp sắp xếp các mô mình đó dựa theo những kết quả trả về qua phép đo đó, rồi từ đó nhanh chóng quyết định mô hình nào hoạt động tốt nhất.
+
 If you really care about both Precision and Recall, I recommend using one of the standard ways to combine them into a single number. For example, one could take the average of precision and recall, to end up with a single number. Alternatively, you can compute the “F1 score,” which is a modified way of computing their average, and works better than simply taking the mean.[4]
 
 ->
 
+Nếu bạn thực sự quan tâm đến cả Precision lẫn Recall. Tôi gợi ý sử dụng một trong những cách tiêu chuẩn để kết hợp các chỉ số đó thành một số duy nhất. Ví dụ, một người có thể lấy giá trị trung bình của Precision và Recall rồi thu về một phép đo đơn trị. Hoặc thay vào đó, bạn có thể tính "chỉ số F1", một cách tính toán sửa đổi của trung bình cộng, và thường hoạt động tốt hơn việc chỉ lấy giá trị trung bình nêu trên.
```

Comment:
'a modified way of computing their average' -> 'một biến thể của trung bình cộng'